hdrass/CCW+Rotator_integration_testing/Rotator_interlock_test.ipynb

```diff
@@ -0,0 +1,1007 @@
+    "script.evt_logMessage.set_put(level=logging.INFO+1,\n",
+    "                              message=f\"START - {test_message} - Move to 1 deg - {now} UTC\")\n",
+    "\n",
+    "print(\"Move to 1 deg position\")\n",
+    "rot.cmd_configureVelocity.set(vlimit=0.02)\n",
+    "await rot.cmd_move.set_start(position=0,timeout=30.)\n",
+    "\n",
+    "print(\"Reset the Rotator and CCW\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Bring the Rotator and CCW back to enabled state"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtm.cmd_clearError.start(timeout=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Wait for Safety Reset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await rot.cmd_clearError.start(timeout=30)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CCWstate = await mtm.evt_cameraCableWrapFollowing.aget(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)\n",
+    "await mtm.cmd_enableCameraCableWrapFollowing.set_start(timeout=10.)\n",
+    "CCWstate = await mtm.evt_cameraCableWrapFollowing.aget(timeout=10.)\n",
+    "print(\"CCWstate:\",CCWstate.enabled)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(rot, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Move to Zero"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Move to 0.0 deg starting position\")\n",
+    "\n",
+    "await rot.cmd_move.set_start(position=0.0,timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.STANDBY)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Wait for override off"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await salobj.set_summary_state(mtm, salobj.State.ENABLED)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set Low Velocity and Move Through Negative Interlock"
+   ]
+  },
+  {
+   "cell_type": "raw",
```

Comment:
Again, no such command. Please don't even try this. It's not in the XML. I can't see how this will do anything except complain that rot has no such attribute cmd_moveConstantVelocity.

Reply:
Taken out.